README.md

```diff
@@ -40,7 +40,7 @@ To build this project, simply run:
 ./gradlew build
 ```
 
-This will build the application and output the resulting files for Windows, Linux and OSX in the `dist` directory.
+This will build the application and output the resulting files for Windows, Linux, and OSX in the `dist` directory.
```

Comment:
Comma here is unnecessary, don't put a comma before and in a list - https://www.grammarly.com/blog/comma-before-and/#:~:text=Comma%20before%20and%20in%20lists